game_calls.cpp

```diff
@@ -69,14 +81,8 @@ bool CheckGuess(std::vector<int> guess, std::vector<int> code)
 	return status;
 }
 
-bool ContainedIn(int current, std::vector<int> code)
+bool ContainedIn(int current, const std::vector<int>& code)
 {
-	for (auto e : code)
-	{
-		if (e == current)
-		{
-			return  true;
-		}
-	}
-	return false;
+	// Let STL do the searching
+	return std::find(code.begin(), code.end(), current) != code.end();
```

Comment:
Okay good to know. Yeah I have briefly used iterators before so makes sense. What is STL?

Reply:
Standard Template Library; basically any `#include <X>` that's part of the standard. (Some are provided by the OS, like `<Windows.h>`.)